app/src/main/java/com/nervousfish/nervousfish/activities/QRExchangeKeyActivity.java

```diff
@@ -99,22 +95,21 @@ private void drawQRCode() {
      * {@inheritDoc}
      */
     @Override
-    public void onActivityResult(final int requestCode, final int resultCode, final Intent intent) {
+    public void onActivityResult(final int requestCode, final int resultCode, final Intent data) {
```

Comment:
I prefer that every argument has the same name as the parameter, and the parameter is named "data" unfortunately